webplugin/gulpfile.js

```
@@ -397,15 +436,72 @@
+        console.log('Source maps successfully uploaded to Sentry.');
+        done();
+    } catch (error) {
+        console.error('Failed to upload source maps:', error);
+    }
+}
+// Task to upload source maps to Sentry
+gulp.task('upload-sourcemaps', function (done) {
+    return new Promise(async (resolve, reject) => {
+        await uploadSourceMaps(done);
+        resolve();
+    });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)